components/ai_chat/core/browser/ai_chat_database.cc

```diff
@@ -146,7 +154,7 @@ constexpr int kLowestSupportedDatabaseVersion = 1;
 constexpr int kCompatibleDatabaseVersionNumber = 1;
```

Comment:
I think we have to increase `kCompatibleDatabaseVersionNumber` when removing a column because older versions of Brave will attempt to insert to or read from the missing column.

Reply:
https://github.com/brave/brave-core/pull/28874/commits/a5644d57c0fedaad0d714e8a8ce7ae45ba451401